app/common/middleware/request.js

```diff
@@ -0,0 +1,14 @@
+const cookies = require('cookie-parser')
+const express = require('express')
+
+/**
+ * Request handling
+ */
+const request = [
+  cookies(),
+
+  // Turn form POSTs into data that can be used for validation
+  express.urlencoded({ extended: true })
```

Comment:
Can we split this change out into a separate commit?

It looks like we no longer need `body-parser` because the functionality is now built into express – is that right? Would be good to capture in the commit message.

Reply:
✅ Done

>It looks like we no longer need body-parser because the functionality is now built into express – is that right? 

Yeah, the `express.urlencoded()` middleware was added to Express.js in v4.16.0
https://expressjs.com/en/api.html#express.urlencoded